apps/showcase/src/components/showcase/component-replacement/component-replacement-pres.style.scss

```diff
@@ -0,0 +1,8 @@
+o3r-component-replacement-pres {
+	// Your component custom SCSS
+  .card {
+    display: flex;
```

Comment:
maybe we could use bootstrap classes instead 
(not mandatory)

Reply:
done :)